datafusion/substrait/Cargo.toml

```diff
@@ -38,9 +38,9 @@ chrono = { workspace = true }
 datafusion = { workspace = true, default-features = true }
 itertools = { workspace = true }
 object_store = { workspace = true }
-pbjson-types = "0.6"
-prost = "0.12"
-substrait = { version = "0.36.0", features = ["serde"] }
+pbjson-types = "0.7"
+prost = "0.13"
+substrait = { version = "0.41", features = ["serde"] }
```

Comment:
need this bump for precision timestamp types to have i64 instead of u64, and for IntervalCompound type

pbjson and prost need to be bumped to match substrait's deps

Reply:
Thanks @Blizzara 

I also have a PR prepared to update a bunch of these dependencies as well queued up for the arrow release next week: https://github.com/apache/datafusion/pull/12032